Example/TABTestKit_ExampleUITests/ExampleFeature.swift

```diff
@@ -22,7 +22,42 @@ class ExampleFeature: TestBase {
       Then(I: seeTheExampleScreen)
     }
   }
+}
+
+class BiometricTests: XCTestCase {
+  
+  private var app: XCUIApplication!
```

Comment:
There's already an `App.shared` you can use so this isn't needed

Reply:
I know, but not sure how to conditionally un-enroll or enroll before the call to `launch()` in `TestBase`, as there's two scenarios and I would like to un-enroll before `test_launchWithBiometricsDisabled` and enroll before `test_biometricsEnabled`. If I don't enrolled before launching then the status won't update. Perhaps I need to sort out the code in the view controller so I can just background and foreground the app so the label gets updated with the new status.